### ISSUE TITLE
os_RealAsinRad and os_RealAcosRad do the opposite thing

### ISSUE THREAD

**Issue by david-why** [CE-Programming/toolchain#382]
When I call `os_RealAsinRad`, it returns the arccos of the input, and when I call `os_RealAcosRad`, it returns the arcsin.

Code:
```c
#include <tice.h>

int main()
{
    real_t num, asin, acos;
    char buf[16];

    num = os_FloatToReal(0.5);
    asin = os_RealAsinRad(&num);
    acos = os_RealAcosRad(&num);

    os_ClrHome();
    os_PutStrFull("asin(0.5)=");
    os_RealToStr(buf, &asin, 0, 0, -1);
    os_PutStrFull(buf);
    os_NewLine();
    os_PutStrFull("acos(0.5)=");
    os_RealToStr(buf, &acos, 0, 0, -1);
    os_PutStrFull(buf);
    os_NewLine();
    os_GetKey();

    return 0;
}
```

Output:
```
asin(0.5)=1.047197551
acos(0.5)=0.5235987756
```
While the expected output is that `asin(0.5)=0.5235987756` and `acos(0.5)=1.047197551`.

Complete code: [asincos.zip](https://github.com/CE-Programming/toolchain/files/8694719/asincos.zip)

**Comment by mateoconlechuga** [comment]
This has already been fixed in 83457c261704c1382891ebddcb12cffc087f3f1e back in February. I would recommend that you download the latest nightly build available here: 

Please read the release note so you can get the latest documentation information too :)

Thanks for the report!

**Comment by mateoconlechuga** [comment]
Update: A new release is available that implements this fix :)
